tripal_chado/src/Form/ChadoPrepareForm.php

```diff
@@ -32,13 +33,43 @@ public function buildForm(array $form, FormStateInterface $form_state) {
         It will also add some management tables to Drupal and add some default
         content types for biological and ancillary data."),
     ];
-
+    
+    $info[] = 'Tripal Chado Integration now supports <strong>setting the schema
+      name</strong> local Chado instances are installed in. In Tripal v3 and
+      lower, the recommended name for your chado schema was <code>chado</code>
+      and that is still the default. Note: Schema name cannot be changed once
+      set.';
```

Comment:
>   Note: Schema name cannot be changed once set.
  
I'm actually not sure this is still true. I think Valentin added a rename chado task.

Reply:
Yeah, it's confusing. I'll rework those instructions both in the install and the prepare forms because I borrowed the one from the prepare from the install.